configuration/configuration.go

```diff
@@ -142,3 +152,74 @@ func GetHTTPAddress() string {
 func IsPostgresDeveloperModeEnabled() bool {
 	return viper.GetBool(varDeveloperModeEnabled)
 }
+
+// GetTokenPrivateKey returns the private key (as set via config file or environment variable)
+// that is used to sign the authentication token.
+func GetTokenPrivateKey() string {
```

Comment:
If you convert the `string` result to `[]byte` anyway to be consumed by `ParseRSAPrivateKeyFromPEM()`, you might as well change the signature of this function to return `[]byte` instead of `string`.